src/app/core/import/edit-import-column-mapping/edit-import-column-mapping.component.spec.ts

```diff
@@ -0,0 +1,76 @@
+import { ComponentFixture, TestBed } from "@angular/core/testing";
+import { EditImportColumnMappingComponent } from "./edit-import-column-mapping.component";
+import { MockedTestingModule } from "app/utils/mocked-testing.module";
+import { MatDialog } from "@angular/material/dialog";
+import { of } from "rxjs";
+import { ColumnMapping } from "../column-mapping";
+import { TestEntity } from "../../../utils/test-utils/TestEntity";
+import { componentRegistry, ComponentRegistry } from "app/dynamic-components";
+import { DiscreteImportConfigComponent } from "app/core/basic-datatypes/discrete/discrete-import-config/discrete-import-config.component";
+
+fdescribe("EditImportColumnMappingComponent", () => {
+  let component: EditImportColumnMappingComponent;
+  let fixture: ComponentFixture<EditImportColumnMappingComponent>;
+  let dialogSpy: jasmine.SpyObj<MatDialog>;
+
+  const columnMapping: ColumnMapping = {
+    column: "test",
+    propertyName: "category",
+  };
+
+  const rawData = [
+    { name: "first", gender: "male" },
+    { name: "second", gender: "female" },
+    { name: "third", gender: "female" },
+  ];
+
+  beforeEach(async () => {
+    dialogSpy = jasmine.createSpyObj("MatDialog", ["open"]);
+    await TestBed.configureTestingModule({
+      imports: [MockedTestingModule, EditImportColumnMappingComponent],
+      providers: [
+        { provide: MatDialog, useValue: dialogSpy },
+        { provide: ComponentRegistry, useValue: componentRegistry },
+      ],
+    }).compileComponents();
+
+    fixture = TestBed.createComponent(EditImportColumnMappingComponent);
+    component = fixture.componentInstance;
+    component.col = columnMapping;
+    component.rawData = rawData;
+    fixture.detectChanges();
+
+    spyOn(component.valueChange, "emit");
+  });
+
```

Comment:
Let's also add one basic test for selecting a mapping.
(feel free to "simulate" the user action by directly calling this.updateMapping() instead of dealing with test harnesses for the html components, though)

```
it("should emit changes after selected entity-field is changed", async () => {
```